File: logica-programacao/python/027.py
# Receba um número.
# Verifique se ele está entre 10 e 20 (inclusive 10 e 20).
# Retorne True ou False.

numero = input("Número: ")
float_numero = float(numero)

verificacao = float_numero >= 10 and float_numero <= 20

print(f"Está entre 10 e 20?: {verificacao}")

File: logica-programacao/javascript/027.js
/* Receba um número.
Verifique se ele está entre 10 e 20 (inclusive 10 e 20).
Retorne True ou False. */

let numero = prompt("Número: ");
let floatNumero = parseFloat(numero);

let verificacao = floatNumero >= 10 && floatNumero <= 20;

console.log(`Está entre 10 e 20?: ${verificacao}`);